docs/api/apiv3/paths/work_schedule_week_days.yml

```diff
@@ -0,0 +1,99 @@
+---
+get:
+  summary: Lists week days
+  operationId: list_week_days
+  tags:
+    - Work Schedule
+    - Non working days
+    - Week days
+  description: |-
+    Lists week days with work schedule information.
+  responses:
+    '200':
+      description: OK
+      content:
+        application/hal+json:
+          schema:
+            $ref: '../components/schemas/week_day_collection_model.yml'
+    '400':
+      description: Returned if the client sends invalid request parameters
+      content:
+        application/hal+json:
+          schema:
+            $ref: '../components/schemas/error_response.yml'
+          example:
+            _type: Error
+            errorIdentifier: urn:openproject-org:api:v3:errors:InvalidRequestBody
+            message: The request body was not a single JSON object.
+
+patch:
+  summary: Update week days
+  operationId: update_week_days
+  tags:
+    - Work Schedule
+    - Non working days
+    - Week days
+  description: |-
+    Update all week days with work schedule information.
+  requestBody:
+    content:
+      application/json:
+        schema:
+          $ref: ../components/schemas/week_day_collection_write_model.yml
+        example:
```

Comment:
no example needed, if provided in the model file

Reply:
Ok. I was not sure about how it would render on the website. Is there a waty to generate the same output as on https://www.openproject.org/docs/api/endpoints ?

Reply:
try http://localhost:4200/api/docs. it is not the same, but same enough ;) There is no polishing in this docs, like we have it at the website in the end.

Reply:
Yep, it is same enough I guess. I'll probably ask Andrej if I need a more accurate one.

Reply:
yep, done.